static/js/zcommand.js

```diff
@@ -159,16 +164,24 @@ exports.process = function (message_content) {
         return true;
     }
 
-    const day_commands = ["/day", "/light"];
-    if (day_commands.includes(content)) {
-        exports.enter_day_mode();
-        return true;
-    }
-
-    const night_commands = ["/night", "/dark"];
-    if (night_commands.includes(content)) {
-        exports.enter_night_mode();
-        return true;
+    const theme_commands = ["/theme"];
```

Comment:
Actually, if you perform the above refactor to extract a `set_theme` function, you could write this part as:

```
if (!['night', 'day', 'auto'].includes(parameter)) return false;
set_theme(parameter);
return true;
```